modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -982,8 +982,11 @@ export class Wallets implements IWallets {
         .result();
       const multisigTypeVersion =
         tssSettings.coinSettings[this.baseCoin.getFamily()]?.walletCreationSettings?.multiSigTypeVersion;
+
       if (this.baseCoin.isEVM() && multisigTypeVersion === 'MPCv2') {
-        walletVersion = 5;
+        if (!walletVersion || (walletVersion !== 5 && walletVersion !== 6)) {
+          walletVersion = 5;
+        }
```

Comment:
can we encapsulate this logic in a function, like `getWalletVersionForMPCv2` ? Just reading this snippet is confusing for someone without context.